### PR TITLE
WebGPU: Depth Texture example

### PR DESCRIPTION
**Pull request by sunag**
Related issue: https://github.com/mrdoob/three.js/issues/23655

**Description**

WebGPU Depth Texture
https://raw.githack.com/sunag/three.js/dev-rtt-webgpu/examples/webgpu_depth_texture.html

This contribution is funded by [Googe via Igalia](https://igalia.com).

### CODE REVIEW

**Comment by sunag** [comment]
@Mugen87 @mrdoob Can I merger this example?

File: examples/webgpu_depth_texture.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 0.01, 30 );
+				camera.position.z = 4;
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0x222222 );
+
+				// depth material
+
+				const material = new Nodes.MeshBasicNodeMaterial();
+				material.colorNode = sub( 1.0, smoothstep( camera.near, camera.far, negate( positionView.z ) ) );
```
Comment (sunag) [inline examples/webgpu_depth_texture.html:66]: Depth Material Approach

Comment (sunag) [inline examples/webgpu_depth_texture.html:66]: Depth Material Approach

File: examples/webgpu_depth_texture.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 0.01, 30 );
+				camera.position.z = 4;
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0x222222 );
+
+				// depth material
+
+				const material = new Nodes.MeshBasicNodeMaterial();
+				material.colorNode = invert( smoothstep( camera.near, camera.far, negate( positionView.z ) ) );
```
Comment (sunag) [inline examples/webgpu_depth_texture.html:66]: Depth Material Approach

**Comment by Mugen87** [comment]
I'm not sure about providing a standalone example with a new depth material. Maybe it's better to focus on `DepthTexture` since in most use cases you want a beauty AND depth pass with a single render.

@mrdoob How do you feel about this?

**Comment by mrdoob** [comment]
Hmm, I think we can keep this for now.
Good to have a few more webgpu examples/tests.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**